configuration/config2/SchemaValue.cpp

```diff
@@ -218,9 +218,14 @@ void SchemaValue::getAllowedValues(std::vector<AllowedValue> &allowedValues, con
 
     //
     // uniqueItemType_1 - value is based on a unique item type described by the data for the rule. This is version 1
-    else if (m_valueLimitRuleType == "uniqueItemType_1")
+    else if (m_valueLimitRuleType == "uniqueItemType_espBinding")
```

Comment:
you might need to update the comment above

Reply:
@rpastrana  Updated.